### DISCUSSION TITLE
What's the best way to use faster-whisper to generate accurate subtitles for `.mp4` videos?

### DISCUSSION THREAD

**Discussion by deadcoder0904** [SYSTRAN/faster-whisper#370] [Q&A]
I use an app called MacWhisper which is great but takes a long-time.

This looks like a faster project.

I would like to use it but haven't seen any cli that generates subtitles. The `whisper-translate2` does look like one but it also does translation.

Is that the one to use?

Are there any better ones? Easier ones? Or maybe just a simple drag & drop one that is not Electron?

**Comment by phineas-pta** [comment]
MacWhisper is built on `whisper.cpp` which has better performance on apple silicon

if u use mac intel then `faster-whisper` should be better

**Reply by deadcoder0904** [reply]
@phineas-pta can i use `faster-whisper` directly through cli? if so, what's the command?

i can't see it on readme.

**Reply by phineas-pta** [reply]
it doesnt have cli, see readme for correct package to use

**Reply by deadcoder0904** [reply]
i went with `whisper-ctranslate2` & it did convert well but `.srt` [generated is in a different format.](https://github.com/Softcatala/whisper-ctranslate2/issues/53)
